labs/b4.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Beginner Lab 4 - Debian, packages, compiling software
+layout: lab
+---
+
+## What is a distribution?
+
+A distribution is a combination of the a kernel, init system, libraries,
+drivers, and other critical software. There are many distributions, for Linux
+and other kernels as well, such as BSD and Solaris.
+
+## What should I use then?
+
+Currently, there are around 300 actively developed Linux distributions listed
+on [DistroWatch](https://distrowatch.com). It might be a bit overwhelming with
+so many choices. Popular distributions include Debian, Ubuntu, Arch Linux,
```

Comment:
Maybe talk a little bit (footnotes) about why they are different instead of just saying they are different.

Reply:
I think I will leave this to you.